src/fetcher.js

```diff
@@ -585,17 +585,18 @@ var Fetcher = function Fetcher (store, timeout, async) {
   // Returns promise of XHR
   //
   //  Writes back to the web what we have in the store for this uri
-  this.putBack = function (uri, options = {}) {
```

Comment:
I thought this came from a linter warning but itactually probably just came from a diff with an older version of the fetcher.